src/flash/com/longtailvideo/jwplayer/controller/Controller.as

```diff
@@ -227,6 +227,13 @@ public class Controller extends GlobalEventDispatcher {
         return false;
     }
 
+    public function init(item:PlaylistItem):void {
+        var _loaded:Boolean = load(item);
```

Comment:
Using underscore as a prefix for variable names is generally used to imply scope - I think the convention used most often in this codebase is to use _ for instance variables which are treated as "private."

So perhaps an alternative would be just ```loaded``` or even better, ```loadSuccessful```